style.css

```diff
@@ -0,0 +1,100 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: "Times", sans-serif;
+}
+
+.wrapper {
+  width: 375px;
+}
```

Comment:
[OPTIONAL] Great job making the changes :clap: , one important thing is our site needs to be responsive on all screen size, and be as close to the figma design as possible currently you have set a fixed width on your wrapper and it looks like :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200119668-9641b874-2828-43ca-b6db-5794060512f3.png)
I would suggest to just simply remove the fixed width, and make the head and headline section take 100vh height so they take up the whole available height you can set the height of your `section.headline{height: 80vh}`, also please adjust the font-size of your `header` links to match the figma design as currently they are very small